localisation/00_zuwc_united_localisation.csv

```diff
@@ -3016,11 +3012,21 @@ EVTOPTB_WCADD_1100;I'll pay [target_crystal_owner.GetHerHim] a visit!;;Ich werde
+DYN6_DESC;With a score of $SCORE$ you$FAILED$ beat house Anvilmar, the descendants of the last High King of all dwarves who ruled the kingdom with justice and wisdom, Modimus Anvilmar. As their society grew too large within the confines of their great mountain, three powerful factions rose among their people: the Bronzebeard, Wildhammer and Dark Iron clans. This divide eventually led to the downfall of the Anvilmar dynasty. When Modimus passed away of old age, his son would not have enough time to be officially coronated before the dwarven civil war had broken out between the three clans.;;Mit einem Punktestand von $SCORE$ ist es Euch$FAILED$ gelungen, das Haus Anvilmar, die Nachkommen des letzten Hochk�nigs aller Zwerge, der das K�nigreich mit Gerechtigkeit und Weisheit regierte, Modimus Anvilmar, zu �bertreffen. Als ihre Gesellschaft innerhalb der Grenzen ihres gro�en Berges zu gro� wurde, erhoben sich drei m�chtige Fraktionen in ihrem Volk: der Bronzebeard-, der Wildhammer- und der Dunkeleisen-Clan. Diese Spaltung f�hrte schlie�lich zum Untergang der Anvilmar-Dynastie. Als Modimus an Altersschw�che starb, hatte sein Sohn nicht genug Zeit, offiziell gekr�nt zu werden, bevor der B�rgerkrieg der Zwerge zwischen den drei Clans ausbrach.;;;;;;;;;;;x
+DYN7_DESC;With a score of $SCORE$ you$FAILED$ beat house Shadowsong. Despite having no noble blood, this family rose to some relevant prominence during the War of the Ancients, with Jarod Shadowsong being a Captain of the Suramar Guard and later the Commander of the Kaldorei Resistance. In the meantime, his sister Maiev was a senior priestess in the Sisters of Elune. After the conflict, she volunteered to become the leader of a newly-formed elite force, the Watchers. The Shadowsong family only reached their prestigious position within Kaldorei society with blood, toil, tears, and sweat.;;Mit einem Punktestand von $SCORE$ ist es Euch$FAILED$ gelungen, das Haus Shadowsong zu �bertreffen. Obwohl diese Familie kein adliges Blut in sich tr�gt, erlangte sie w�hrend des Krieges der Ahnen eine gewisse Bedeutung, als Jarod Shadowsong Hauptmann der Suramar-Garde und sp�ter Kommandant des Kaldorei-Widerstandes war. In der Zwischenzeit war seine Schwester Maiev als Hohepriesterin bei den Schwestern von Elune t�tig. Nach dem Konflikt meldete sie sich freiwillig, um die Anf�hrerin einer neu gebildeten Elitetruppe, der W�chter, zu werden. Die Familie Shadowsong erreichte ihre prestigetr�chtige Position innerhalb der Kaldorei-Gesellschaft nur mit Blut, M�hsal, Tr�nen und Schwei�.;;;;;;;;;;;x
```

Comment:
Standardizing capitalized House in `beat House xxx`
```suggestion
DYN7_DESC;With a score of $SCORE$ you$FAILED$ beat House Shadowsong. Despite having no noble blood, this family rose to some relevant prominence during the War of the Ancients, with Jarod Shadowsong being a Captain of the Suramar Guard and later the Commander of the Kaldorei Resistance. In the meantime, his sister Maiev was a senior priestess in the Sisters of Elune. After the conflict, she volunteered to become the leader of a newly-formed elite force, the Watchers. The Shadowsong family only reached their prestigious position within Kaldorei society with blood, toil, tears, and sweat.;;Mit einem Punktestand von $SCORE$ ist es Euch$FAILED$ gelungen, das Haus Shadowsong zu �bertreffen. Obwohl diese Familie kein adliges Blut in sich tr�gt, erlangte sie w�hrend des Krieges der Ahnen eine gewisse Bedeutung, als Jarod Shadowsong Hauptmann der Suramar-Garde und sp�ter Kommandant des Kaldorei-Widerstandes war. In der Zwischenzeit war seine Schwester Maiev als Hohepriesterin bei den Schwestern von Elune t�tig. Nach dem Konflikt meldete sie sich freiwillig, um die Anf�hrerin einer neu gebildeten Elitetruppe, der W�chter, zu werden. Die Familie Shadowsong erreichte ihre prestigetr�chtige Position innerhalb der Kaldorei-Gesellschaft nur mit Blut, M�hsal, Tr�nen und Schwei�.;;;;;;;;;;;x
```

Reply:
thx again